shared/wallets/transaction/index.tsx

```diff
@@ -309,24 +301,24 @@ const roleToColor = (role: Types.Role): string => {
   }
 }
 
-const getAmount = (role: Types.Role, amountXLM: string): string => {
+const getAmount = (role: Types.Role, amount: string, sourceAmount?: string): string => {
   switch (role) {
     case 'senderOnly':
-      return `- ${amountXLM}`
+      return `- ${sourceAmount || amount}`
     case 'airdrop':
     case 'receiverOnly':
-      return `+ ${amountXLM}`
+      return `+ ${amount}`
     case 'senderAndReceiver':
       return '0 XLM'
```

Comment:
I'm not sure if `0 XLM` is the best label for when you're both the sender and receiver. It leads to transactions like these:
![Screen Shot 2019-07-01 at 4 02 44 PM](https://user-images.githubusercontent.com/5677971/60463329-ae115e00-9c19-11e9-97d2-87d7a9bff5ce.png)
which is technically correct, but doesn't have anything to do with lumens.